src/commons/editor/Editor.tsx

```diff
@@ -485,8 +482,16 @@ const EditorBase = React.memo((props: EditorProps & LocalStateProps) => {
       if (!reactAceRef.current) {
         return;
       }
-      handleEditorValueChange(props.editorTabIndex, newCode);
+
       shiftBreakpointsWithCode(reactAceRef.current.editor, delta);
+
+      // Write editor state for the active editor tab to the store.
+      handleEditorValueChange(props.editorTabIndex, newCode);
+      handleEditorUpdateBreakpointsRef.current(
```

Comment:
No idea why this callback function is wrapped in a ref but I'm afraid that removing it will cause some other regression somewhere.